samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -369,6 +372,9 @@ static int lwm2m_setup(void)
 	lwm2m_firmware_set_write_cb(firmware_block_received_cb);
 #endif
 #if defined(CONFIG_LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT)
+	lwm2m_engine_create_res_inst("5/0/8/0");
+	lwm2m_engine_set_res_data("5/0/8/0", &supported_protocol, sizeof(supported_protocol), 0);
```

Comment:
Each resource instance should point to a single uint8_t value, there it should rather take `&supported_protocol[0]` and `sizeof(supported_protocol[0])`, otherwise, if someone decides to extend the array you'll provide invalid data to this resource instance.